alcs-frontend/src/app/features/application/application.component.ts

```diff
@@ -24,15 +25,20 @@ export const childRoutes = [
     menuTitle: 'Processing',
     component: ProcessingComponent,
   },
+  {
+    path: 'site-visit-applicant',
```

Comment:
This should probably be `site-visit-meeting`?

Reply:
Changed!